test/integration/tls/tls_app/tls/certs.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
Where is this script used?

Reply:
Script is only required if someone needs to regenerate the certs. Will add a comment.